src/components/CardDetails.js

```diff
@@ -0,0 +1,270 @@
+    ) {
+      setCharacterType("/dokkanIcons/types/teq-background.png");
+      setCharacterSuperOrExtreme("/dokkanIcons/types/eteq.png");
+    } else if (
+      cardDetails.type === "STEQ" ||
+      cardDetails.type === "STEQ " ||
+      cardDetails.type === "TEQ-S"
+    ) {
+      setCharacterType("/dokkanIcons/types/teq-background.png");
+      setCharacterSuperOrExtreme("/dokkanIcons/types/steq.png");
+    }
+  }, [characterId]);
+
+  function handleSaveCharacter() {
+    const username = Auth.getProfile().data.username;
+    saveCharacter({
+      variables: {
+        username: username,
+        dokkanId: cardDetails.id,
+      },
+    }).then((result) => {
+      console.log(result);
+    });
+    handleCardDetailsReload(cardDetails);
+  }
+
+  function handleRemoveCharacter() {
+    const username = Auth.getProfile().data.username;
+    removeCharacter({
+      variables: {
+        username: username,
+        dokkanId: cardDetails.id,
+      },
+    }).then((result) => {
+      console.log(result);
+    });
+     handleCardDetailsReload(cardDetails);
+  }
+
+  return (
+    <div className="grid grid-cols-1 place-items-center">
+      <div className="bg-gradient-radial from-purple-200 via-purple-100 to-purple-50 border-2 border-black m-1 p-2 shadow-[inset_0_-5px_6px_rgba(0,0,0,0.6)] border-2 border-black w-max rounded-lg">
+        <h2 className="ml-1 mt-2 text-center text-4xl">{cardDetails.name}</h2>
+      </div>
+      <div>
+        <div>
+          {characterThumb && (
+            <div className="relative">
+              <div
+                onClick={() => {}}
+                className="h-48 w-48 m-2 gap-4 bg-no-repeat relative z-50"
+                style={{
+                  backgroundImage: `url("https://dokkan.wiki/assets/global/en/character/thumb/card_${characterThumb}_thumb.png")`,
+                  backgroundSize: `100%`,
+                }}
+              ></div>
+              <img className={characterRarityPosition} src={characterRarity} />
+              <img
+                className="w-40 absolute top-[15px] right-[25px] z-0"
+                src={characterType}
+              />
+              <img
+                className="w-20 absolute top-[-12px] right-[4px] z-50"
+                src={characterSuperOrExtreme}
+              />
+            </div>
+          )}
+        </div>
+      </div>
+      <div className="bg-gradient-radial from-purple-200 via-purple-100 to-purple-50 m-1 p-2 shadow-[inset_0_-5px_6px_rgba(0,0,0,0.6)] border-2 border-slate-900 w-[85%] h-min">
+        <p>Leader Skill: {cardDetails.ls_description}</p>
+      </div>
+
+      <div className="bg-gradient-radial from-purple-200 via-purple-100 to-purple-50 m-1 p-2 shadow-[inset_0_-5px_6px_rgba(0,0,0,0.6)] border-2 border-slate-900 w-[85%] h-min">
+        <p>
+          {cardDetails.sa_name}: {cardDetails.sa_description}
+        </p>
+      </div>
+      <div>
+            {Auth.loggedIn() ? (
+              <div>
```

Comment:
Looks like this div (and the same in the other branch on line 228) can be deleted